docs/config/_default/params.yaml

```diff
@@ -0,0 +1,10 @@
+github_branch: main
+github_project_repo: https://github.com/keptn/lifecycle-toolkit
+github_repo: https://github.com/keptn/lifecycle-toolkit
+github_subdir: docs
+
+versions:
+- url: /docs/
+  version: development
+- url: https://lifecycle.keptn.sh
+  version: production
```

Comment:
missing newline :)

Reply:
https://github.com/keptn/lifecycle-toolkit/issues/904 - i hate it if there is something tooling can cover and we need to manually take care of it :)